pkg/ovncontroller/volumes.go

```diff
@@ -0,0 +1,117 @@
+package ovncontroller
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+// GetVolumes - Volumes used by pod
+func GetVolumes(cmName string) []corev1.Volume {
+	var scriptsVolumeDefaultMode int32 = 0755
+	return []corev1.Volume{
+		{
+			Name: "host-run-netns",
+			VolumeSource: corev1.VolumeSource{
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: "/run/netns",
+				},
+			},
+		},
+		{
+			Name: "run-openvswitch",
+			VolumeSource: corev1.VolumeSource{
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: "/run/openvswitch",
+				},
+			},
+		},
+		{
+			Name: "etc-openvswitch",
+			VolumeSource: corev1.VolumeSource{
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: "/etc/openvswitch",
+				},
+			},
+		},
+		{
+			Name: "var-lib-openvswitch",
+			VolumeSource: corev1.VolumeSource{
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: "/var/lib/openvswitch",
+				},
+			},
+		},
+		{
+			Name: "env-overrides",
+			VolumeSource: corev1.VolumeSource{
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: "/env",
```

Comment:
what can we control with the info in /env/${K8S_NODE}? I see it gets sourced in the script. Is it for debugging?